### PR TITLE
Improvements from opentopomap.org

### PR DESCRIPTION
**Pull request by der-stefan**
This pull request contains three commits:
- add default style for easy usage
- fix bug #1 (delete event after deleting the grid layer)
- add option on redrawing (`move` or `moveend`)

### CODE REVIEW

File: L.Grid.js [hunk 1]
```diff
@@ -20,7 +20,10 @@ L.Grid = L.LayerGroup.extend({
 			color: '#111',
 			opacity: 0.6,
 			weight: 1
-		}
+		},
+		
+		// Redraw on move or moveend
+		redraw: 'move'
```
Comment (der-stefan) [inline L.Grid.js:26]: I don't know if it is necessary to check for wrong values. Let's trust the user?

**Comment by jieter** [comment]
Hmm, nice it's actually useful to someone. Thanks!

**Comment by der-stefan** [comment]
My wish is to draw a grid with 1x1km on higher zoom levels, so that you can print out the map and use it offline with your GPS. Right now experimenting with lat/lon in degree for lower zoom levels is nice to have... ;-)

**Comment by der-stefan** [comment]
By the way: You could add Leaflet.Grid to the official Leaflet plugin list! See how it works on the very bottom of this page: http://leafletjs.com/plugins.html
(I would have liked to write you a private message for this, but unluckily Github canceled this feature)

**Comment by jieter** [comment]
Thanks for the suggestion, but I did not add it because I think it's not polished enough yet...

**Pull request merged**